Sources/NIOHTTPCompression/HTTPDecompression.swift

```diff
@@ -46,7 +46,7 @@ public enum NIOHTTPDecompression {
         }
     }
 
-    public enum DecompressionError: Error {
+    public enum DecompressionError: Error, Equatable {
```

Comment:
👀 @Lukasa This makes the tests a little easier to read. If this is an unwanted api change, I'm happy to revert it.